account_reconcile_payment_order/__openerp__.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#    This module copyright (C) 2015 Therp BV <http://therp.nl>.
+#
+#    This program is free software: you can redistribute it and/or modify
+#    it under the terms of the GNU Affero General Public License as
+#    published by the Free Software Foundation, either version 3 of the
+#    License, or (at your option) any later version.
+#
+#    This program is distributed in the hope that it will be useful,
+#    but WITHOUT ANY WARRANTY; without even the implied warranty of
+#    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#    GNU Affero General Public License for more details.
+#
+#    You should have received a copy of the GNU Affero General Public License
+#    along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+##############################################################################
+{
+    "name": "Reconcile payment orders",
+    "version": "1.0",
+    "author": "Therp BV,Odoo Community Association (OCA)",
+    "license": "AGPL-3",
+    "category": "Accounting & Finance",
+    "summary": "Automatically reconcile all lines from sent payment orders",
+    "depends": [
+        'account_banking_payment_transfer',
```

Comment:
Thanks for the clarification. Then, definitively the module name should be _account_bank_statement_reconcile_payment_order_. We can't remove the bank_statement part from the name because we have several kind of reconciliations, and this one is for bank statements.